cmd/main.go

```diff
@@ -257,6 +259,7 @@ func (opt *Options) bindCLIFlagsToViper(f *pflag.FlagSet) error {
 	f.IntVar(&opt.MaxIterations, "max-iterations", opt.MaxIterations, "maximum number of iterations agent will try before giving up")
 	f.StringVar(&opt.KubeConfigPath, "kubeconfig", opt.KubeConfigPath, "path to kubeconfig file")
 	f.StringVar(&opt.PromptTemplateFilePath, "prompt-template-file-path", opt.PromptTemplateFilePath, "path to custom prompt template file")
+	f.StringArrayVar(&opt.AdditionalPromptTemplateFilePaths, "additional-prompt-template-file-paths", opt.AdditionalPromptTemplateFilePaths, "additional paths to custom prompt template files")
```

Comment:
The name is getting super verbose :) One option is to name it `extra-prompt-paths` (extra replaces additional pretty well)

Thinking out loud here: if the paths point to a directory, we can slurp all files from that directory (this could be added later as well doesn't have to be this PR).

PS: We will introduce short flags soon to help with the verbosity.